package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@dsnp/activity-content",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LibertyDSNP/activity-content.git"
+  },
+  "license": "ISC",
```

Comment:
We also need a license file like: https://github.com/LibertyDSNP/example-client/blob/main/LICENSE

```suggestion
  "license": "Apache-2.0",
```

Reply:
Added license file